cornflow/endpoints/instance.py

```diff
@@ -117,17 +122,55 @@ def delete(self, idx):
         return self.delete_detail(self.user_id, idx)
 
 
-class InstanceFileEndpoint(MetaResource):
+class InstanceFileEndpoint(InstanceEndpoint):
     """
     Endpoint to accept mps files to upload
     """
 
-    def post(self, file):
+    @Auth.auth_required
+    def post(self):
         """
-        Not implemented!
 
         :param file:
         :return:
         :rtype: Tuple(dict, integer)
         """
-        return {}, 501
+        self.user_id, self.admin, self.super_admin = Auth.return_user_info(request)
```

Comment:
How is this form actually sent?

Reply:
You can check the [cornflow-client tests](https://github.com/baobabsoluciones/corn/blob/feature/airflowStatus_and_mpsFile/cornflow/tests/unit/test_cornflowclient.py) or the [flask tests](https://github.com/baobabsoluciones/corn/blob/feature/airflowStatus_and_mpsFile/cornflow/tests/unit/test_instances_file.py).